Cargo.lock

```diff
@@ -1992,9 +1992,9 @@ dependencies = [
 
 [[package]]
 name = "lyon_geom"
-version = "1.0.16"
+version = "1.0.17"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "ce9333c02ea4517fd31207f126124352ad59975218c114c55dbb8f9d56fd4b45"
+checksum = "4e16770d760c7848b0c1c2d209101e408207a65168109509f8483837a36cf2e7"
 dependencies = [
```

Comment:
Even though IIRC we saw in the Ruffle repo that this could change a couple pixels here and there (due to different tessellation, see https://github.com/ruffle-rs/ruffle/pull/21758), and we held it back there to not make the visual tests fail (https://github.com/ruffle-rs/ruffle/pull/21764), I think it will be just fine to bump it here.